components/authz-service/server/v2/policy.go

```diff
@@ -140,7 +146,7 @@ func (s *policyServer) CreatePolicy(
 
 	// API requests always create custom policies.
 
-	pol, err := policyFromAPI(
+	pol, err := s.policyFromAPI(
```

Comment:
I'm curious why the change to have this be a function on `server` . . . maybe it's too late for me to be reviewing? 🙃

Reply:
No, that's a fair question! It is (almost) turtles all the way down. In other words, at the bottom, we need to access the `vSwitch` which is on `s`. Follow that back up to here: `s.vSwitch` => s.isBeta2p1 => s.statementFromAPI => s.policyFromAPI.

Reply:
🤔 Is that a smell? 🤷‍♂